### ISSUE TITLE
Add documentation on how to use L function and i18n tag in Pug

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#607]
### Problem

It's unclear what the rules are for using the `L` function in Pug.

### Solution

Add documentation showing how and when to use the `L` function, along with `v-html` [as shown here](https://github.com/okTurtles/group-income-simple/pull/606/commits/f846ae8dead1ce7f1083ed3f2ad278876364d123) for dealing with markup.

Basically, `i18n` should be used except when:

**UPDATE: Some of the stuff written below is outdated! See [this pull request for latest usage rules](https://github.com/okTurtles/group-income-simple/pull/727)**

- Passing in string literals as props to a component (e.g. `page-section(:title='L("foo")')`)
- String needs to include markup, like `<b>foo</b>`, in which case ~~use `v-html`~~ (see edit below)
- `template` is used, in which case you need to use it like this:

   ```pug
   template(#title='') {{ L('Group Settings') }}
   
   //- OR like this, but note this inserts an extra <span> tag
   template(#title='')
     i18n Group Settings
   ```

EDIT: this has been changed in PR #611, see [comment](https://github.com/okTurtles/group-income-simple/pull/606/commits/f846ae8dead1ce7f1083ed3f2ad278876364d123#r313655392), copied here:

> NOTE: I have changed this (since the above doesn't work very well in my experience, especially not with variables as well), in my PR #611 — now the way to this is simple:
> 
> ```pug
> i18n(tag='p' html='This means you will stop having access to the <b>group chat</b> (including direct messages to other group members) and <b>contributions</b>. Re-joining the group is possible, but requires other members to vote and reach an agreement.')
> ```
> 
> **EDIT**: the above is now invalid! There should be no more use of the `html=` attribute. **See [this PR for how to do it now](https://github.com/okTurtles/group-income-simple/pull/727)**

**EDIT:** see if using [literal HTML in Pug](https://pugjs.org/language/plain-text.html#literal-html) makes this any cleaner/easier.
